### ISSUE TITLE
Compiled with warning

### ISSUE THREAD

**Issue by apasov** [quasarframework/quasar-ui-qiconpicker#26]
After upgrading to @quasar/quasar-app-extension-qiconpicker v1.0.10 I get this error when compiling in dev mode:
```
 WARNING  Compiled with 1 warnings                                                                                                                                                                                                   17:26:53

 warning  in ./node_modules/@quasar/quasar-ui-qiconpicker/dist/index.esm.js

Module not found: Error: Can't resolve 'q-icon-picker/components/icon-set' in 'c:\my-app\node_modules\@quasar\quasar-ui-qiconpicker\dist'

i ｢wds｣: Project is running at http://0.0.0.0:8088/
i ｢wds｣: webpack output is served from
i ｢wds｣: 404s will fallback to /index.html
```

The project compiles okay and icon picker works as intended but this warning confuses me.

**Comment by tomers** [comment]
I got this:
```

 ERROR  Failed to compile with 1 errors                                                                                                               5:50:50 PM

This dependency was not found:

* api in ./node_modules/babel-loader/lib??ref--1-0!./node_modules/vue-loader/lib??vue-loader-options!./src/pages/Index.vue?vue&type=script&lang=js&

To install it, you can run: npm install --save api
```

**Comment by hawkeye64** [comment]
API is a webpack alias. It works. Not sure why you're getting warnings. I'm not seeing them. Make sure you build the UI folder first.

**Comment by apasov** [comment]
@hawkeye64 I really don't understand why should I build the UI folder or whatever else. It's just a patch version change, isn't it? There should be no inconsistencies at all when upgrading from 1.0.9 to 1.0.10
Why nothing is mentioned about this in release message?
I just searched @quasar/quasar-ui-qiconpicker/dist/index.esm.js for "icon-set" and found it 4 times:
```
n=require("@quasar/quasar-ui-qiconpicker/src/components/icon-set/"+i+".js")
o=require("../src/components/icon-set/"+i+".js")
a=require("./icon-set/"+i+".js")
s=require("q-icon-picker/components/icon-set/"+i+".js")
```
Obviously, it is the last line which triggers the warning.

**Comment by hawkeye64** [comment]
QIconPicker (`@quasar/quasar-ui-qiconpicker v1.0.6` and `@quasar/quasar-app-extension-qiconpicker v1.0.11`) have been released.
https://github.com/quasarframework/quasar-ui-qiconpicker/releases/tag/%40quasar%2Fquasar-ui-qiconpicker-v1.0.6
- fix: Compiled with warning (#26)
--------
Sorry for the issues, was entirely my fault. Originally, I thought it was entirely something else.
